share/search/index_strategy/trovesearch_denorm.py

```diff
@@ -154,15 +155,14 @@ def _paths_and_values_mappings(self):
 
     # override method from Elastic8IndexStrategy
     def after_chunk(self, messages_chunk: messages.MessagesChunk, indexnames: Iterable[str]):
-        # refresh to avoid delete-by-query conflicts
-        self.es8_client.indices.refresh(index=','.join(indexnames))
-        # delete any docs that belong to cards in this chunk but weren't touched by indexing
-        self.es8_client.delete_by_query(
-            index=list(indexnames),
-            query={'bool': {'must': [
-                {'terms': {'card.card_pk': messages_chunk.target_ids_chunk}},
-                {'range': {'chunk_timestamp': {'lt': messages_chunk.timestamp}}},
-            ]}},
+        task__delete_iri_value_scraps.apply_async(
+            kwargs={
+                'index_strategy_name': self.name,
+                'indexnames': list(indexnames),
+                'card_pks': messages_chunk.target_ids_chunk,
+                'timestamp': messages_chunk.timestamp,
+            },
+            countdown=3,  # TODO: config?
```

Comment:
Minor: Should probably be configurable, yes.
Something like 
`settings.ELASTICSEARCH['COUNTDOWN_DELAY'] = int(os.environ.get('ELASTICSEARCH_COUNTDOWN_DELAY', 3))`
would probably be reasonable

Reply:
went with `ELASTICSEARCH_POST_INDEX_DELAY`